docs/dev/tools/common-errors.md

```diff
@@ -184,27 +184,11 @@ This can happen when there is a slow internet connection, when the module is try
 **Solution:**
 
 - Try using a faster internet connection.
-- If the problem persists, try increasing the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables in the module's JSON configuration file.
-  For example:
-
-  ```json {class="line-numbers linkable-line-numbers"}
-  {
-    "modules": [
-      {
-        ...
-        "env": {
-          "VIAM_MODULE_STARTUP_TIMEOUT": "6m30s",
-          "VIAM_RESOURCE_CONFIGURATION_TIMEOUT": "3m0s"
-        }
-      }
-    ]
-  }
-  ```
-
 - If you are the module author, consider packaging the module with required dependencies so they don't need to be downloaded on startup.
   For Python modules, you can package your module with dependencies by using the PyInstaller steps when [uploading your module](/operate/get-started/other-hardware/#upload-your-module).
-
-See [Module Configuration Details](/operate/reference/module-configuration/#environment-variables) for more information on these environment variables.
+- If the problem persists, try setting the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables on your machine using the `viam-server` service configuration file.
+  Pass a sequence of numbers and time units, for example "6m30s50ms" for a timeout of 6 minutes, 30 seconds, and 50 milliseconds, or "5m" for a timeout of 5 minutes.
```

Comment:
Ooh by a full example I meant like `VIAM_MODULE_STARTUP_TIMEOUT=6m30 viam-server -config xxx.json` or `Environment="VIAM_MODULE_STARTUP_TIMEOUT=6m30s"` (for each of the two  current ways to do this). Do you think we shouldn't show the command line way?

Reply:
yep, i think we're on the same wavelength -- while I would _like_ to show the command line way, it requires a bit too much explanation so I think we're safer distancing ourselves from it a bit. Especially since this is a temporary situation and we'll hopefully have the JSON configuration back as best practice in a bit.

Reply:
Seems like from what Cheuk said if we are only going to show one way it should be the command line way and not the service file way, but we could show both

Reply:
But also yes temporary so I don't suppose it matters!

Reply:
Thanks for this feedback, I added a CLI example showing how to configure this during manual `viam-server` startup. Assuming our checks pass, I'll merge this in so we don't have any more outdated info in the docs. This solution isn't perfect, but we know we'll update this eventually to demonstrate a better solution anyway. Good enough for this interim period.